config/config.go

```diff
@@ -8,55 +8,55 @@ import (
 )
 
 type Config struct {
-	Path                  string       `json:"path" yaml:"path" mapstructure:"path" `
-	AppID                 string       `json:"appId" yaml:"appId" mapstructure:"appId"`
+	Path                  string       `json:"path" yaml:"path" mapstructure:"path"`
```

Comment:
See if this change is not needed because you are binding camelCase flags to viper